### PR TITLE
Add social open graph meta to page header

### PR DESCRIPTION
**Pull request by bdemers**
This should provide social previews to the sdkman site when sharing links in various places (Twitter, FB, Slack, etc)

### CODE REVIEW

File: app/views/includes/header.scala.html [hunk 1]
```diff
@@ -9,4 +9,15 @@
     <link rel='stylesheet' type='text/css' href='@routes.Assets.at("/css/font-awesome.min.css")'/>
     <link rel='stylesheet' type='text/css' href='@routes.Assets.at("/css/style.css")'/>
     <link rel='stylesheet' type='text/css' href='@routes.Assets.at("/css/carbon.css")'/>
+
+    <!-- Social bits -->
+    <meta property='og:type' content='website' />
+    <meta property='og:url' content='https://sdkman.io/' />
```
Comment (bdemers) [inline app/views/includes/header.scala.html:15]: Ideally, this should probably be the absolute URL to the current page, is there a `@` variable for this?

If not, it's not the end of the world 🤷

Comment (marc0der) [reply app/views/includes/header.scala.html:15]: No, but we should be able to compose this. I'll see if I can get it to work on the weekend.

Comment (marc0der) [reply app/views/includes/header.scala.html:15]: @bdemers I've managed to propagate this through the request and will add a commit to compose the URL.

**Comment by marc0der** [comment]
Thanks for contributing this! I'll release your changes soon.

**Comment by bdemers** [comment]
Nice!! Thanks for wrapping this up @marc0der!

**Pull request merged**